samples/src/main/java/org/corfudb/samples/HeloCorfu.java

```diff
@@ -0,0 +1,84 @@
+package org.corfudb.samples;
+
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.collections.SMRMap;
+
+import java.util.Map;
+
+/**
+ * This tutorial demonstrates a simple Corfu application.
+ *
+ * Created by dalia on 12/30/16.
+ */
+public class HeloCorfu {
+
+    static String defaultCorfuService = "localhost:9999";
```

Comment:
This isn't set anywhere... and args aren't used. Shouldn't this be set by the command line?